powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/shared/PowerAuthCallbackShared.java

```diff
@@ -69,6 +70,11 @@ public static void callbackExecutionTest(PowerAuthClient powerAuthClient, PowerA
         request.put("blockedReason", "TEST_CALLBACK");
         request.put("applicationId", config.getApplicationId());
         RestClientFactory.getRestClient().post(callbackUrlVerify, request, new ParameterizedTypeReference<String>() {});
+        try {
+            Thread.sleep(1000);
```

Comment:
🤔 It could be fine to replace `Thread.sleep`. For example something like
```java
Awaitility
    .await()
    .pollDelay(Duration)
    .untilAsserted(() -> TODO);
```

Reply:
Please raise a separate issue, this is not a point of this change.